Doc/library/math.rst

```diff
@@ -192,6 +207,18 @@ Number-theoretic and representation functions
    of *x* and are floats.
 
 
+.. function:: perm(n, k)
+
+   Return the number of ways to choose *k* items from *n* items without repetition.
```

Comment:
The "with order" part should be mentioned.  Perhaps ``s/the number/the distinct number/``.

FWIW, here is the wording from the *itertools* docs:


```
permutations() | p[, r] | r-length tuples, all possible orderings, no repeated elements

combinations() | p, r | r-length tuples, in sorted order, no repeated elements
```